cmd/history/history.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Extract git history from the main kata repos and generate two report files:
+#  report.txt - commits merged, sorted by repo section, and date.
+#  sorted_report.txt - all commits sorted by date
+#
+# This information can be useful when tracking down 'what changed when'.
+
+set -e
+
+REPORTFILE="report.txt"
+SORTEDREPORTFILE="sorted_${REPORTFILE}"
+
+# Set a date to start from, or use the default of '1 week'
+# Format is in any acceptible by 'Date -I -d'.
+defstart="last week"
```

Comment:
Worth making these `defstart=${defstart:-last week}` for consistency with the others maybe?

Reply:
sure. I didn't as I was thinking folks would override the `START` etc., but no harm, so adding.